resources/gobos/glass-black-10-circles.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/gobo.json",
+  "name": "Black with 10 circles",
+  "keywords": "glass dots",
```

This is not a glass gobo, but a metal one.

```suggestion
  "keywords": "dots",
```